app/models/user.rb

```diff
@@ -42,7 +42,7 @@ def self.select_status_list
   end
 
   def self.filter_status
-    status_nice_to_meet_you
+    [status_nice_to_meet_you, status_available]
```

Comment:
Could this be a variable like we have done with the `VALID_STATUS_LIST`?

Reply:
I think this is already enough abstraction